TMDbLib/Client/TMDbClientTrending.cs

```diff
@@ -18,6 +19,9 @@ public async Task<SearchContainer<SearchMovie>> GetTrendingMoviesAsync(TimeWindo
             if (page >= 1)
                 req.AddQueryString("page", page.ToString());
 
+            if (!string.IsNullOrWhiteSpace(DefaultLanguage))
```

Comment:
https://github.com/jellyfin/TMDbLib/pull/427 has been merged. Can you update your PR?